prisma/schema.prisma

```diff
@@ -266,6 +266,26 @@ model Seeded {
   @@ignore
 }
 
+enum SwishOrderStatus {
+  PAID
+  DECLINED
+  ERROR
+  CANCELLED
+}
+
+model SwishOrder {
+  ID              Int               @id @default(autoincrement())
+  KID             String            @db.VarChar(15)
+  instructionUUID String            @db.VarChar(36) /// Swish uses this to identify the order
+  donorID         Int
+  donationID      Int?
+  status          SwishOrderStatus? /// External status from swish
+  reference       String            @db.VarChar(32)
+  registered      DateTime          @default(now()) @db.DateTime(0)
+
+  @@index([donorID], map: "Donor_ID_idx")
```

Comment:
Is this just a normal index or a foreign key to the donors table?
If not an FK, probably should have one, no?

Reply:
This is a leftover from the `Vipps_orders` model. It seems like `SwishOrder` is queried by `ID` or `instructionUUID`. Perhaps this current index makes no sense and we should add one for `instructionUUID` instead?

Reply:
Aha, yes, that'd be good! But a foreign key to donorID would be good as well?
Let's say we change a donor ID, we want the changes to cascade through the model.
I would lean towards:
Change -> Cascade
Delete -> Sett null (to preserve history for data purposes)

Reply:
Might be nice to preserve history, although seems like donor deletion would cascade to donations, which would then cascade to swish orders. So regardless of using `SetNull` or `Delete` behaviour on the SwishOrder <> Donor relation, it would still be deleted, right?